src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.cmc.claimstore.documents;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.events.DocumentReadyToPrintEvent;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.sendletter.api.Letter;
+import uk.gov.hmcts.reform.sendletter.api.SendLetterApi;
+
+@Service
+@ConditionalOnProperty(prefix = "send-letter", name = "url")
+public class BulkPrintService {
```

Comment:
Would be nice to have a unit test for this service.

Reply:
what would it do and what benefit would it bring?

Reply:
I think I would add a unit test which checks that the printing api is given all the documents of the event (checks that the list is not being modified) and a Spring test which verifies that this listener is invoked upon submitting a claim (checks that business logic is in place).